test/cri-containerd/container_layers_packing_test.go

```diff
@@ -0,0 +1,81 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+const ubuntu1804 = "ubuntu:18.04"
+
+func filterStrings(input []string, include string) []string {
+	var result []string
+	for _, str := range input {
+		if strings.Contains(str, include) {
+			result = append(result, str)
+		}
+	}
+	return result
+}
+
+func shimDiag(t *testing.T, podID string, cmd []string) string {
+	shimID := fmt.Sprintf("k8s.io-%s", podID)
+	args := append([]string{"exec", shimID}, cmd...)
+	sc := exec.Command("shimdiag.exe", args...)
+	out, err := sc.Output()
+	if err != nil {
+		t.Errorf("shimdiag error: %s", err)
+	}
+	return string(out)
+}
+
+func validateTargets(t *testing.T, podID string, expected int) {
+	dmDiag := shimDiag(t, podID, []string{"ls", "-l", "/dev/mapper"})
```

Comment:
Can't we use `execSync` in our test suite instead of invoking shimdiag?

Reply:
TIL good point.

Reply:
I might have been looking at a wrong thing, but it doesn't look like `execSync` can exec commands inside the UVM?

Reply:
Oh crud, I didn't read this as deeply as I thought I apologize. Yes ExecSync only works for containers, do we really not have a way/wrapper to exec a command in the uvm easily in these tests?

Reply:
Was just talking with @katiewasnothere about this after I read your reply

Reply:
I think it's fine to just do what Maksim is doing here tbh. I've made some tests in the past for device things that also relied on calling test binaries, so I don't think there's anything wrong with it.

Reply:
I don't either, I just thought we had some wrapper already for this but I can't seem to find one. Maybe I'll push something to add this